docs/mergeduty/howto.md

```diff
@@ -150,44 +150,25 @@ push: true
 1. Follow the [general steps](#general-steps)
 1. Insert the following payload and click submit.
 
-```
+```yaml
 force-dry-run: true
 merge_flavor: central-to-beta
 push: true
 ```
 
 #### mozilla-esr bump no-op trial run
 
-#### ESR bump requires a different copy of mozharness
-
-You need to download a different set of mozharness because ESR68 has a different configuration due to Fennec ESR.
 
-```sh
-export version=68
-export full_version=68.4.0
-mkdir ~/merge_day_esr_${full_version}
-cd ~/merge_day_esr_${full_version}
-wget -O mozharness.zip https://hg.mozilla.org/releases/mozilla-esr${version}/archive/tip.zip/testing/mozharness/
-unzip mozharness.zip
-mv mozilla-esr${version}-*/testing/mozharness . && rmdir mozilla-esr${version}-*/testing
-wget -O mozbase.zip https://hg.mozilla.org/releases/mozilla-esr${version}/archive/tip.zip/testing/mozbase/
-unzip mozbase.zip
-mv mozilla-esr${version}-*/testing/mozbase . && rmdir mozilla-esr${version}-*/testing
-rm mozbase.zip mozharness.zip
-for package in manifestparser mozinfo mozprocess mozfile; do cp -pr mozbase/${package}/${package} mozharness/; done
- ```
-
-Run the bump-esr [no-op trial run](#do-migration-no-op-trial-runs), and show the diff to another person on releaseduty.
+1. Follow the [general steps](#general-steps)
```

Comment:
Nit: Should we add a word about going on `mozilla-central` instead of `mozilla-esr68`, until the patch gets uplifted?

Reply:
Good plan! Done